tdvt/tdvt/exprtests/standard/expected.setup.cast.str.datetime.1.txt

```diff
@@ -0,0 +1,120 @@
+<results>
```

Comment:
Should this file be called `tdvt/tdvt/exprtests/standard/expected.setup.cast.str.datetime.txt`?

Reply:
We have 2 separate expected files

Reply:
Thanks I missed that.

Please regenerate `expected.setup.cast.str.datetime.txt` using hyper, then we can keep `expected.setup.cast.str.datetime.1.txt` as-is. (I assume hyper will have the same results.) 

For reference, which ds did we use to generate the second set of expecteds? It's confusing since both expecteds say `cast_calcs.sqlserver`.

Reply:
I generated expected.setup.cast.str.datetime.txt using hyper. Second set is for datasources where second value could not be converted. It is based on sqlserver.